examples/AzureSDKDemoSwift/Source/BlobTableViewController.swift

```diff
@@ -160,7 +160,7 @@ class BlobTableViewController: UITableViewController, MSALInteractiveDelegate {
                 showAlert(error: "Unable to create URL!")
                 return
             }
-            self.showActivitySpinner()
+            showActivitySpinner()
```

Comment:
It seems like swiftformat is not picking up the config file, these changes shouldn't be showing up. If you just run `swiftformat sdk` from the root of the repo, does it revert these changes?

Reply:
You can also explicitly pass the config file path with `--config <repo_root>/.swiftformat` but that shouldn't be necessary.

Reply:
I think swiftformat is working, because I ran a build and it made a bunch of changes I had to commit. This was likely swiftformat undoing parts that may have been reintroduced by a rebase.